src/algebra/module/basic.lean

```diff
@@ -613,7 +613,7 @@ end module
 
 section division_ring
 
-variables [division_ring R] [add_comm_group M] [module R M]
+variables [division_ring R] [add_comm_monoid M] [module R M]
```

Comment:
This is surprising to me: a module over a division ring is an additive group. It's not so much that this is wrong for technical reasons, but it does suggest we're missing something where we need it.

Reply:
Sorry for the lack of update, this has been further improved and merged in #16673. This change was originally made for replacing the lemma `aeval_eq_zero_of_aeval_algebra_map_eq_zero_field`.